README.md

```diff
@@ -151,7 +151,6 @@ Commands:
 
 - `rustic-compile`            compile project using `rustic-compile-command`
 - `rustic-recompile`          recompile using `compilation-arguments`
-- `rustic-compile-send-input` send string to process of current buffer
```

Comment:
I'd like to keep this function since it allows you to send input to the process even if you are using the compilation mode.

Reply:
@brotzeit Thanks for the review. I have restored the function and have also restored the polymode based functionality. I have added the details in the README on how it differss.